src/Behat/Behat/Gherkin/ServiceContainer/GherkinExtension.php

```diff
@@ -79,25 +79,26 @@ public function initialize(ExtensionManager $extensionManager)
      */
     public function configure(ArrayNodeDefinition $builder)
     {
-        $builder
+        $childrenBuilder = $builder
```

Comment:
In some cases the `end()` function will return a type that cannot be used to chain other calls to other config functions because these other functions are not defined in the returned type. We split the chain of calls so that the functions are always defined in the returned types

Reply:
IMO this is also more readable so 👍